test/parallel/test-crypto-keygen.js

```diff
@@ -1543,44 +1543,65 @@ const sec1EncExp = (cipher) => getRegExpForPEM('EC PRIVATE KEY', cipher);
   }
 }
 
-if (!common.hasOpenSSL3) {
-  // Passing an empty passphrase string should not cause OpenSSL's default
-  // passphrase prompt in the terminal.
-  // See https://github.com/nodejs/node/issues/35898.
-
-  for (const type of ['pkcs1', 'pkcs8']) {
-    generateKeyPair('rsa', {
-      modulusLength: 1024,
-      privateKeyEncoding: {
-        type,
-        format: 'pem',
-        cipher: 'aes-256-cbc',
-        passphrase: ''
-      }
-    }, common.mustSucceed((publicKey, privateKey) => {
-      assert.strictEqual(publicKey.type, 'public');
+// Passing an empty passphrase string should not cause OpenSSL's default
```

Comment:
I'd prefer to keep the indentation and make it a block like elsewhere in this test and others. Smaller diff and I like the pattern to separate independent tests, and because blocks can be collapsed in most editors.

```suggestion
{
  // Passing an empty passphrase string should not cause OpenSSL's default
```

Reply:
@tniessen isn't the test present in a block already (the for loop)?

> Smaller diff

Did you try reviewing the change with `w=1` (hides whitespace changes) as in https://github.com/nodejs/node/pull/42319/files?diff=split&w=1?

> I like the pattern to separate independent tests,

Is it really worth maintaining the individual blocks per tests this way, given that it can get out of sync very easily since we don't enforce the rule using a linter? I noticed that the usage of blocks in the test suite is already inconsistent.

> and because blocks can be collapsed in most editors.

Is that often required for test files?

Reply:
Yeah, I guess the loop does make it a block :)